media-gfx/curaengine/curaengine-4.7.0.ebuild

```diff
@@ -0,0 +1,88 @@
+# Copyright 1999-2020 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=7
+
+inherit cmake toolchain-funcs
+
+MY_PN="CuraEngine"
+
+DESCRIPTION="A 3D model slicing engine for 3D printing"
+HOMEPAGE="https://github.com/Ultimaker/CuraEngine"
+SRC_URI="https://github.com/Ultimaker/${MY_PN}/archive/${PV}.tar.gz -> ${P}.tar.gz"
+
+LICENSE="AGPL-3"
+SLOT="0"
+KEYWORDS="~amd64 ~x86"
+IUSE="+arcus doc openmp test"
+RESTRICT="!test? ( test )"
+
+RDEPEND="
+	${PYTHON_DEPS}
+	arcus? (
+		~dev-libs/libarcus-${PV}:*
+		dev-libs/protobuf:=
+	)
+	dev-libs/clipper
+	dev-libs/rapidjson
+	dev-libs/stb"
+
+DEPEND="${RDEPEND}
```

Comment:
doc? at least is probably BDEPEND, as it needs to be executed

Reply:
Correct, I've fixed that